### PR TITLE
build: upgrade Vue

### PR DESCRIPTION
**Pull request by ArturLange**
### What

Upgrade `vue` for dynamic import of components to work in #349 

v3.3.9 --> v3.4.19

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1123,15 +1172,15 @@ vue@^3.2.25:
     "@vue/shared" "3.3.13"
 
 vue@^3.3.9:
```
Comment (raphodn) [inline yarn.lock:1174]: which command did you run ? `yarn upgrade` right ?

maybe change the the `package.json` as well to reflect the change ? `^3.4.19`

https://github.com/vuejs/core/blob/main/CHANGELOG.md

Comment (ArturLange) [reply yarn.lock:1174]: I used `yarn upgradeInteractive`. 
`3.4.19` fulfills `^3.3.9` so it should be valid.

Comment (raphodn) [reply yarn.lock:1174]: Yet the version change would just be cosmetic ^^

Comment (ArturLange) [reply yarn.lock]: Done then :slightly_smiling_face:

**Review by raphodn — APPROVED** [review]
:+1:

**Pull request merged**